active/NumberBox/NumberBox.md

```diff
@@ -0,0 +1,220 @@
+# Number box
+
+<!-- The purpose of this spec is to describe a new feature and
+its APIs that make up a new feature in WinUI. -->
+
+<!-- There are two audiences for the spec. The first are people
+that want to evaluate and give feedback on the API, as part of
+the submission process.  When it's complete
+it will be incorporated into the public documentation at
+docs.microsoft.com (http://docs.microsoft.com/uwp/toolkits/winui/).
+Hopefully we'll be able to copy it mostly verbatim.
+So the second audience is everyone that reads there to learn how
+and why to use this API. -->
+
+## Background
+<!-- Use this section to provide background context for the new API(s) 
+in this spec. -->
+
+<!-- This section and the appendix are the only sections that likely
+do not get copied to docs.microsoft.com; they're just an aid to reading this spec. -->
+
+<!-- If you're modifying an existing API, included a link here to the
+existing page(s) -->
+
+<!-- For example, this section is a place to explain why you're adding this API rather than
+modifying an existing API. -->
+
+<!-- For example, this is a place to provide a brief explanation of some dependent
+area, just explanation enough to understand this new API, rather than telling
+the reader "go read 100 pages of background information posted at ...". -->
+
+
+## Description
+<!-- Use this section to provide a brief description of the feature.
+For an example, see the introduction to the PasswordBox control 
+(http://docs.microsoft.com/windows/uwp/design/controls-and-patterns/password-box). -->
+
+Number box is a numerics only text control with support for input formatting, validation, basic calculation, and increment stepping.
+
+**Important APIs:** [NumberBox class](https://docs.microsoft.com/en-us/uwp/api/microsoft.ui.xaml.controls.numberbox)
+
+## Is this the right control? 
+
+Use a **NumberBox** control to recieve and display formatted numeric values such as currency, IP addresses, and telephone numbers. A **NumberBox** can also be enabled with basic calculator support to compute multiplication, division, addition, and subtraction across order of operations on entered equations. 
+
+## Examples
+<!-- Use this section to explain the features of the API, showing
+example code with each description. The general format is: 
+  feature explanation,
+  example code
+  feature explanation,
+  example code
+  etc.-->
+  
+<!-- Code samples should be in C# and/or C++/WinRT -->
+
+<!-- As an example of this section, see the Examples section for the PasswordBox control 
+(https://docs.microsoft.com/windows/uwp/design/controls-and-patterns/password-box#examples). -->
+
+### Create a NumberBox
+
+XAML
+```XAML
+<NumberBox AutomationProperties.Name="simple NumberBox"/>
```

Comment:
Why have you specified `AutomationProperties.Name` instead of just `Name` in these examples?
Do you mean to call attention to something specific about automation support? If so, what?

Regardless, the automation property names used in this spec serve as bad examples as automation names should not include the name of the control.

Reply:
You're right, @mrlacey, this was a bite from the copy paste bug w/ regard to our Xaml Controls Gallery TextBox sample. I appreciate @YuliKl, our Accessibility Champion, for educating me further on automation properties and their redundancy with Headers on TextBox and its derivatives.

Reply:
Resolved here: https://github.com/microsoft/microsoft-ui-xaml-specs/commit/20ba7ae9254674200d8f2d363bc08949533c37b4